tasks-app-shared/src/commonMain/kotlin/net/opatry/tasks/app/di/networkModule.kt

```diff
@@ -56,10 +57,18 @@ val networkModule = module {
         val credentialsStorage = get<CredentialsStorage>()
 
         HttpClient(CIO) {
-            CurlUserAgent()
             install(ContentNegotiation) {
                 json()
             }
+            install(UserAgent) {
```

Comment:
If properly confirmed content is gzipped this way, try using `CurlUserAgent()` without `"gzip"` like that and check it still works.

Could also check nothing is gzipped if `install(ContentEncoding)` is removed.